app/src/main/org/runnerup/view/StartActivity.java

```diff
@@ -327,6 +328,9 @@ public void handleOnBackPressed() {
                 }
               }
             });
+
+    mWearNotifier = new TrackerWear.WearNotifier(getApplicationContext());
+    mWearNotifier.onCreate();

```

Comment:
Changed with fragments in #1233 
```suggestion
    mWearNotifier.onViewCreated();

```